### PR TITLE
chore(localdebug): add debug config file into tab sso v3 template

### PR DESCRIPTION
**Pull request by XiaofuHuang**
Add debug into tab sso v3 template

### CODE REVIEW

**Comment by swatDong** [comment]
Why adding an empty `.env.local`?
- Could just remove it since no value there?
- Or, add some placeholders and comments (similar to .env.dev)?

**Review by swatDong — DISMISSED** [review]

**Comment by XiaofuHuang** [comment]
> Why adding an empty `.env.local`?
> 
> * Could just remove it since no value there?
> * Or, add some placeholders and comments (similar to .env.dev)?

Now if `apply` api will throw error if `.env` file does not exist. So the empty .env.local file make the f5 work.

I added some placeholders and comments in .env.local.

**Comment by kuojianlu** [comment]
Should we also update the placeholder for launch remote, e.g. `dev:teamsAppId`?

**Comment by XiaofuHuang** [comment]
> Should we also update the placeholder for launch remote, e.g. `dev:teamsAppId`?

Now v3 will have a question to choose env.
image

Will remote show the same question?

**Review by swatDong — APPROVED** [review]

File: templates/scenarios/sso-tab-js/.vscode/tasks.json [hunk 1]
```diff
@@ -2,45 +2,72 @@
...
-            "label": "prepare local environment",
-            "type": "shell",
-            "command": "exit ${command:fx-extension.pre-debug-check}",
-            "presentation": {
-                "reveal": "never"
+            "label": "Create resources",
+            "type": "teamsfx",
+            "command": "provision",
+            "args": {
+                "configFile": "${workspaceFolder}/teamsfx/app.local.yml",
```
Comment (kuojianlu) [inline templates/scenarios/sso-tab-js/.vscode/tasks.json:34]: -> `template`

Comment (XiaofuHuang) [reply templates/scenarios/sso-tab-js/.vscode/tasks.json:34]: Will updated in PR: https://github.com/OfficeDev/TeamsFx/pull/6578

**Pull request merged**